api/openapi/v2/core-metadata.yaml

```diff
@@ -122,8 +107,8 @@ components:
       properties:
         deviceServiceName:
           type: string
-        addressableName:
-          type: string
+        addressable:
+          $ref: '#/components/schemas/Addressable'
```

Comment:
It was agreed to remove `Addressable`. So need new properties that replace it, could simply be `Url` or `Address` or multiple values that build the address such as Protocol, Host, Port, Path. I personally prefer `Url` or `Address`.

Reply:
Currently, we simplify the addressable as the following struct but still need discussion with community.
.
type Addressable struct {
	~Timestamps~
	~Id          string `json:"id,omitempty"`          // ID is a unique identifier for the Addressable, such as a UUID~
	~Name        string `json:"name,omitempty"`        // Name is a unique name given to the Addressable~
	Protocol    string `json:"protocol,omitempty"`    // Protocol for the address (HTTP/TCP)
	~HTTPMethod  string `json:"method,omitempty"`      // Method for connecting (i.e. POST)~
	Address     string `json:"address,omitempty"`     // Address of the addressable
	Port        int    `json:"port,omitempty,Number"` // Port for the address
	Path        string `json:"path,omitempty"`        // Path for callbacks
	~Publisher   string `json:"publisher,omitempty"`   // For message bus protocols~
	User        string `json:"user,omitempty"`        // User id for authentication
	Password    string `json:"password,omitempty"`    // Password of the user for authentication for the addressable
	~Topic       string `json:"topic,omitempty"`       // Topic for message bus addressables~
}

Reply:
FYI in the Core WG meeting today is was decided to replace the Addressable property with BaseAddress with is a fully qualified URI string.

Reply:
Modified. Replace the addressable with baseAddress.